Add log configuration in coredns docs (#323)
giantswarm/docs: src/content/guides/advanced-coredns-configuration/index.md

## Cache settings

> By default we set the cache TTL for CoreDNS to 30 seconds. You can customize the cache settings of CoreDNS by setting the value of the cache field in the user ConfigMap like following:

Reviewer: ```suggestion
By default we set the cache TTL for CoreDNS to 30 seconds. You can customize the cache settings of CoreDNS by setting the value of the cache field in the user ConfigMap like this:
```

Minor but flows better for me.

The cache plugin also supports much more detailed configuration which is documented in the [upstream documentation](https://coredns.io/plugins/cache/).

## Logs

> By default, we set the log level for CoreDNS to `denial` and `error`. You can tun these settings by adding a property `log` in the user ConfigMap like following:

Reviewer: ```suggestion
By default, we set the log level for CoreDNS to `denial` and `error`. You can tun these settings by adding a property `log` in the user ConfigMap like this:
```

```yaml
data:
  log: 
  - all
```

> For knowing the exact details of each log level log plugin, please read the [upstream documentation](https://coredns.io/plugins/log/).

Reviewer: ```suggestion
To know the exact details of each log level log plugin, please read the [upstream documentation](https://coredns.io/plugins/log/).
```
